packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -33,6 +33,7 @@ function render(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchF
         <Item key="campaigns">Campaigns</Item>
         <Item key="audiences">Audiences</Item>
         <Item key="tags">Tags</Item>
+        <Item key="long">This item is very long and word wraps poorly</Item>
```

Comment:
Note: IMO the width of the SearchField element should be reduced for the SearchWithin case. As it is now, the picker can only grow so much since the component will strive to make the SearchField as large as possible, restricting the SearchWithin Picker to 72px:
![image](https://user-images.githubusercontent.com/9661127/128099216-06197a34-bebe-4752-8112-c0479100285a.png)

Lemme know if this is as per Spectrum design tho

Reply:
Yeah this seems weird.

Reply:
cc @reidbarber @snowystinger just wanna run the above by you both since I think the respective SearchField vs Picker widths were discussed in a Spectrum design review previously? We can always revisit in a follow PR if need be

Reply:
for the width of the textfield, i think it's reasonable for it to have a smaller default/min width (that's what you're suggesting right?)
it should probably be single-line-input-width (whatever that's called) - var(--spectrum-global-dimension-size-900) (the min-width of the picker)

Reply:
I'm leaning more towards increasing the picker's min-width since it is the one with flex-shrink